docs/app/api_spec.md

```diff
@@ -0,0 +1,111 @@
+# API Specification: Assistive Chatbot Integration
+This API enables the integration of an Assistive Chatbot with other products.
+
+## Endpoint: `POST /chatbot`
```

Comment:
Can this be a verb instead?
```suggestion
## Endpoint: `POST /query`
```

Reply:
Definitely. What about `chat` though -- slightly more specific than `query`. (I don't feel strongly -- go for whatever you think is best.)

Reply:
Oh wait we already have `/chat` for the frontend, nevermind 🤦

Reply:
Yup!